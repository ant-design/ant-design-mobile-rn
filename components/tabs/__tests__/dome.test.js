import rnDemoTest from '../../../tests/shared/demoTest'

rnDemoTest('tabs')
