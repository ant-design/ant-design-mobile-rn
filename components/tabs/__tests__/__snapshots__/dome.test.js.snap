// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/tabs/demo/basic.tsx correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        Object {},
      ]
    }
  >
    <View
      style={
        Object {
          "borderBottomColor": "#dddddd",
          "borderBottomWidth": 1,
        }
      }
    >
      <View
        onLayout={[Function]}
        style={
          Array [
            Object {},
            Object {
              "backgroundColor": "#fff",
            },
          ]
        }
      >
        <ScrollView
          bounces={false}
          directionalLockEnabled={true}
          horizontal={true}
          renderToHardwareTextureAndroid={true}
          scrollEnabled={false}
          scrollsToTop={false}
          showsHorizontalScrollIndicator={false}
          showsVerticalScrollIndicator={false}
        >
          <View
            onLayout={[Function]}
            style={
              Array [
                Object {
                  "backgroundColor": "#ffffff",
                  "elevation": 0,
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "space-around",
                  "shadowOpacity": 0,
                  "shadowRadius": 0,
                },
                Object {
                  "backgroundColor": "#fff",
                },
              ]
            }
          >
            <
              accessibilityTraits="button"
              accessible={true}
              activeOpacity={1}
              onLayout={[Function]}
              onPress={[Function]}
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "height": 42,
                    "justifyContent": "center",
                    "padding": 0,
                    "width": 250,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": undefined,
                        "fontSize": 15,
                      },
                      Object {},
                    ]
                  }
                >
                  First Tab
                </Text>
              </View>
            </>
            <
              accessibilityTraits="button"
              accessible={true}
              activeOpacity={1}
              onLayout={[Function]}
              onPress={[Function]}
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "height": 42,
                    "justifyContent": "center",
                    "padding": 0,
                    "width": 250,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": undefined,
                        "fontSize": 15,
                      },
                      Object {},
                    ]
                  }
                >
                  Second Tab
                </Text>
              </View>
            </>
            <
              accessibilityTraits="button"
              accessible={true}
              activeOpacity={1}
              onLayout={[Function]}
              onPress={[Function]}
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                    "height": 42,
                    "justifyContent": "center",
                    "padding": 0,
                    "width": 250,
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "color": undefined,
                        "fontSize": 15,
                      },
                      Object {},
                    ]
                  }
                >
                  Third Tab
                </Text>
              </View>
            </>
            <View
              style={
                Object {
                  "backgroundColor": "#108ee9",
                  "bottom": 0,
                  "height": 2,
                  "left": 0,
                  "position": "absolute",
                  "width": 0,
                }
              }
            />
          </View>
        </ScrollView>
      </View>
    </View>
    <RNCViewPager
      initialPage={0}
      keyboardDismissMode="on-drag"
      onMoveShouldSetResponderCapture={[Function]}
      onPageScroll={[Function]}
      onPageScrollStateChanged={[Function]}
      onPageSelected={[Function]}
      scrollEnabled={true}
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={false}
        style={
          Array [
            Object {
              "width": 750,
            },
            Object {
              "bottom": 0,
              "height": undefined,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": undefined,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "height": 150,
              "justifyContent": "center",
            }
          }
        >
          <Text>
            Content of First Tab
          </Text>
        </View>
      </View>
      <View
        collapsable={false}
        style={
          Array [
            Object {
              "width": 750,
            },
            Object {
              "bottom": 0,
              "height": undefined,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": undefined,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "height": 150,
              "justifyContent": "center",
            }
          }
        >
          <Text>
            Content of Second Tab
          </Text>
        </View>
      </View>
      <View
        collapsable={false}
        style={
          Array [
            Object {
              "width": 750,
            },
            Object {
              "bottom": 0,
              "height": undefined,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": undefined,
            },
          ]
        }
      />
    </RNCViewPager>
  </View>
  <Text
    style={
      Object {
        "margin": 16,
      }
    }
  >
    Custom RenderTabBar
  </Text>
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        Object {},
      ]
    }
  >
    <View
      style={
        Object {
          "borderBottomColor": "#dddddd",
          "borderBottomWidth": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "space-evenly",
            "paddingHorizontal": 16,
          }
        }
      >
        <
          activeOpacity={0.9}
          onPress={[Function]}
          style={
            Object {
              "padding": 6,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "green",
              }
            }
          >
            First Tab
          </Text>
        </>
        <
          activeOpacity={0.9}
          onPress={[Function]}
          style={
            Object {
              "padding": 6,
            }
          }
        >
          <Text
            style={
              Object {
                "color": undefined,
              }
            }
          >
            Second Tab
          </Text>
        </>
        <
          activeOpacity={0.9}
          onPress={[Function]}
          style={
            Object {
              "padding": 6,
            }
          }
        >
          <Text
            style={
              Object {
                "color": undefined,
              }
            }
          >
            Third Tab
          </Text>
        </>
      </View>
    </View>
    <RNCViewPager
      initialPage={0}
      keyboardDismissMode="on-drag"
      onMoveShouldSetResponderCapture={[Function]}
      onPageScroll={[Function]}
      onPageScrollStateChanged={[Function]}
      onPageSelected={[Function]}
      scrollEnabled={true}
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={false}
        style={
          Array [
            Object {
              "width": 750,
            },
            Object {
              "bottom": 0,
              "height": undefined,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": undefined,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "height": 150,
              "justifyContent": "center",
            }
          }
        >
          <Text>
            Content of First Tab
          </Text>
        </View>
      </View>
      <View
        collapsable={false}
        style={
          Array [
            Object {
              "width": 750,
            },
            Object {
              "bottom": 0,
              "height": undefined,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": undefined,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "height": 150,
              "justifyContent": "center",
            }
          }
        >
          <Text>
            Content of Second Tab
          </Text>
        </View>
      </View>
      <View
        collapsable={false}
        style={
          Array [
            Object {
              "width": 750,
            },
            Object {
              "bottom": 0,
              "height": undefined,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": undefined,
            },
          ]
        }
      />
    </RNCViewPager>
  </View>
  <View
    style={
      Object {
        "flex": 2,
      }
    }
  >
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flex": 1,
          },
          Object {},
        ]
      }
    >
      <View
        style={
          Object {
            "borderBottomColor": "#dddddd",
            "borderBottomWidth": 1,
          }
        }
      >
        <View
          onLayout={[Function]}
          style={
            Array [
              Object {},
              Object {
                "backgroundColor": "#fff",
              },
            ]
          }
        >
          <ScrollView
            bounces={false}
            directionalLockEnabled={true}
            horizontal={true}
            renderToHardwareTextureAndroid={true}
            scrollEnabled={true}
            scrollsToTop={false}
            showsHorizontalScrollIndicator={false}
            showsVerticalScrollIndicator={false}
          >
            <View
              onLayout={[Function]}
              style={
                Array [
                  Object {
                    "backgroundColor": "#ffffff",
                    "elevation": 0,
                    "flex": 1,
                    "flexDirection": "row",
                    "justifyContent": "space-around",
                    "shadowOpacity": 0,
                    "shadowRadius": 0,
                  },
                  Object {
                    "backgroundColor": "#fff",
                  },
                ]
              }
            >
              <
                accessibilityTraits="button"
                accessible={true}
                activeOpacity={1}
                onLayout={[Function]}
                onPress={[Function]}
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                      "height": 42,
                      "justifyContent": "center",
                      "padding": 0,
                      "width": 150,
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": undefined,
                          "fontSize": 15,
                        },
                        Object {},
                      ]
                    }
                  >
                    1st Tab
                  </Text>
                </View>
              </>
              <
                accessibilityTraits="button"
                accessible={true}
                activeOpacity={1}
                onLayout={[Function]}
                onPress={[Function]}
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                      "height": 42,
                      "justifyContent": "center",
                      "padding": 0,
                      "width": 150,
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": undefined,
                          "fontSize": 15,
                        },
                        Object {},
                      ]
                    }
                  >
                    2nd Tab
                  </Text>
                </View>
              </>
              <
                accessibilityTraits="button"
                accessible={true}
                activeOpacity={1}
                onLayout={[Function]}
                onPress={[Function]}
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                      "height": 42,
                      "justifyContent": "center",
                      "padding": 0,
                      "width": 150,
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": undefined,
                          "fontSize": 15,
                        },
                        Object {},
                      ]
                    }
                  >
                    3rd Tab
                  </Text>
                </View>
              </>
              <
                accessibilityTraits="button"
                accessible={true}
                activeOpacity={1}
                onLayout={[Function]}
                onPress={[Function]}
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                      "height": 42,
                      "justifyContent": "center",
                      "padding": 0,
                      "width": 150,
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": undefined,
                          "fontSize": 15,
                        },
                        Object {},
                      ]
                    }
                  >
                    4th Tab
                  </Text>
                </View>
              </>
              <
                accessibilityTraits="button"
                accessible={true}
                activeOpacity={1}
                onLayout={[Function]}
                onPress={[Function]}
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                      "height": 42,
                      "justifyContent": "center",
                      "padding": 0,
                      "width": 150,
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": undefined,
                          "fontSize": 15,
                        },
                        Object {},
                      ]
                    }
                  >
                    5th Tab
                  </Text>
                </View>
              </>
              <
                accessibilityTraits="button"
                accessible={true}
                activeOpacity={1}
                onLayout={[Function]}
                onPress={[Function]}
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                      "height": 42,
                      "justifyContent": "center",
                      "padding": 0,
                      "width": 150,
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": undefined,
                          "fontSize": 15,
                        },
                        Object {},
                      ]
                    }
                  >
                    6th Tab
                  </Text>
                </View>
              </>
              <
                accessibilityTraits="button"
                accessible={true}
                activeOpacity={1}
                onLayout={[Function]}
                onPress={[Function]}
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                      "height": 42,
                      "justifyContent": "center",
                      "padding": 0,
                      "width": 150,
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": undefined,
                          "fontSize": 15,
                        },
                        Object {},
                      ]
                    }
                  >
                    7th Tab
                  </Text>
                </View>
              </>
              <
                accessibilityTraits="button"
                accessible={true}
                activeOpacity={1}
                onLayout={[Function]}
                onPress={[Function]}
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                      "height": 42,
                      "justifyContent": "center",
                      "padding": 0,
                      "width": 150,
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": undefined,
                          "fontSize": 15,
                        },
                        Object {},
                      ]
                    }
                  >
                    8th Tab
                  </Text>
                </View>
              </>
              <
                accessibilityTraits="button"
                accessible={true}
                activeOpacity={1}
                onLayout={[Function]}
                onPress={[Function]}
              >
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "flexDirection": "row",
                      "height": 42,
                      "justifyContent": "center",
                      "padding": 0,
                      "width": 150,
                    }
                  }
                >
                  <Text
                    style={
                      Array [
                        Object {
                          "color": undefined,
                          "fontSize": 15,
                        },
                        Object {},
                      ]
                    }
                  >
                    9th Tab
                  </Text>
                </View>
              </>
              <View
                style={
                  Object {
                    "backgroundColor": "#108ee9",
                    "bottom": 0,
                    "height": 2,
                    "left": 0,
                    "position": "absolute",
                    "width": 0,
                  }
                }
              />
            </View>
          </ScrollView>
        </View>
      </View>
      <RNCViewPager
        initialPage={1}
        keyboardDismissMode="on-drag"
        onMoveShouldSetResponderCapture={[Function]}
        onPageScroll={[Function]}
        onPageScrollStateChanged={[Function]}
        onPageSelected={[Function]}
        scrollEnabled={true}
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <View
          collapsable={false}
          style={
            Array [
              Object {
                "width": 750,
              },
              Object {
                "bottom": 0,
                "height": undefined,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": undefined,
              },
            ]
          }
        >
          <ScrollView
            style={
              Object {
                "backgroundColor": "#fff",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                1st Tab
                 - 
                1
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                1st Tab
                 - 
                2
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                1st Tab
                 - 
                3
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                1st Tab
                 - 
                4
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                1st Tab
                 - 
                5
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                1st Tab
                 - 
                6
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                1st Tab
                 - 
                7
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                1st Tab
                 - 
                8
              </Text>
            </View>
          </ScrollView>
        </View>
        <View
          collapsable={false}
          style={
            Array [
              Object {
                "width": 750,
              },
              Object {
                "bottom": 0,
                "height": undefined,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": undefined,
              },
            ]
          }
        >
          <ScrollView
            style={
              Object {
                "backgroundColor": "#fff",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                2nd Tab
                 - 
                1
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                2nd Tab
                 - 
                2
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                2nd Tab
                 - 
                3
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                2nd Tab
                 - 
                4
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                2nd Tab
                 - 
                5
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                2nd Tab
                 - 
                6
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                2nd Tab
                 - 
                7
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                2nd Tab
                 - 
                8
              </Text>
            </View>
          </ScrollView>
        </View>
        <View
          collapsable={false}
          style={
            Array [
              Object {
                "width": 750,
              },
              Object {
                "bottom": 0,
                "height": undefined,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": undefined,
              },
            ]
          }
        >
          <ScrollView
            style={
              Object {
                "backgroundColor": "#fff",
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                3rd Tab
                 - 
                1
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                3rd Tab
                 - 
                2
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                3rd Tab
                 - 
                3
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                3rd Tab
                 - 
                4
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                3rd Tab
                 - 
                5
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                3rd Tab
                 - 
                6
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                3rd Tab
                 - 
                7
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ddd",
                  "justifyContent": "center",
                  "margin": 10,
                  "paddingVertical": 40,
                }
              }
            >
              <Text>
                3rd Tab
                 - 
                8
              </Text>
            </View>
          </ScrollView>
        </View>
        <View
          collapsable={false}
          style={
            Array [
              Object {
                "width": 750,
              },
              Object {
                "bottom": 0,
                "height": undefined,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": undefined,
              },
            ]
          }
        />
        <View
          collapsable={false}
          style={
            Array [
              Object {
                "width": 750,
              },
              Object {
                "bottom": 0,
                "height": undefined,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": undefined,
              },
            ]
          }
        />
        <View
          collapsable={false}
          style={
            Array [
              Object {
                "width": 750,
              },
              Object {
                "bottom": 0,
                "height": undefined,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": undefined,
              },
            ]
          }
        />
        <View
          collapsable={false}
          style={
            Array [
              Object {
                "width": 750,
              },
              Object {
                "bottom": 0,
                "height": undefined,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": undefined,
              },
            ]
          }
        />
        <View
          collapsable={false}
          style={
            Array [
              Object {
                "width": 750,
              },
              Object {
                "bottom": 0,
                "height": undefined,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": undefined,
              },
            ]
          }
        />
        <View
          collapsable={false}
          style={
            Array [
              Object {
                "width": 750,
              },
              Object {
                "bottom": 0,
                "height": undefined,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": undefined,
              },
            ]
          }
        />
      </RNCViewPager>
    </View>
  </View>
</View>
`;
